tests/filecheck/dialects/vector/vector_ops_mlir_conversion.xdsl

```diff
@@ -0,0 +1,22 @@
+// RUN: xdsl-opt -t mlir %s | filecheck %s
```

Comment:
I am fine with this approach as well, considering that you are not using any custom prints.

I was thinking of actually passing something to `mlir-opt`, then print and parse again with xDSL.